sopel/config/core_section.py

```diff
@@ -949,13 +949,20 @@ def homedir(self):
 
     """
 
-    modes = ValidatedAttribute('modes', default='B')
+    modes = ValidatedAttribute('modes')
     """User modes to be set on connection.
 
-    :default: ``B``
-
     Include only the mode letters; this value is automatically prefixed with
     ``+`` before Sopel sends the MODE command to IRC.
+
+    .. versionchanged:: 8.0.0
+
+        Now empty by default. Previous default was ``B``, which has been
+        dropped in favor of the formal `bot mode specification`__.
+
+
```

Comment:
One empty line should be enough, right? 😁 

```suggestion

```

Reply:
Whoops, lol. Thanks. 😅